applications/kodadot_assethub_nft_m2.md

```diff
@@ -0,0 +1,224 @@
+
+## Team :busts_in_silhouette:
+
+### Team members (In order of joining time)
+
+- Matej Nemček - Project Lead
+- Viktor Valaštín - Developer
+
+### Contact 📞
+
+- **Contact Name:** Viktor Valaštín
+- **Contact Email:** viktorko99@gmail.com
+
+### Legal Structure
+
+- **Registered Address:** Ljubljanska cesta 4, 4260 Bled, Slovenia, Europe
+- **Registered Legal Entity:** KodaLabs, Viktor Valaštín s.p.
+
+### Team's experience
+
+
+**Viktor Valaštín**, also known as Viki Val, is the Co-founder of KodaDot. He is responsible for the technical aspects of the project. Viktor has been working on implementing MoonBeam and MoonRiver NFT EVM smart contracts and enabling read-only access to existing components for seamless end-user interaction. His technical expertise has been crucial in successfully launching the Basilisk NFT Marketplace pallet in the Fall of 2022.
+
+**Matej Nemček** is the Founder and CEO of [KodaDot](https://kodadot.xyz/). He has been instrumental in the growth and development of KodaDot, leading the team to create the best end-user experience on the Asset hub. Matej's leadership and vision have been pivotal in transforming KodaDot into a collaborative hub where creators, developers, and community members work collectively for decision-making.
+
+Matej and Viktor are strongly committed to the Polkadot ecosystem and have demonstrated their ability to deliver high-quality, impactful projects. They bring a wealth of knowledge and experience to the AssetHub Indexer project. Their work has earned KodaDot the number one rank as a decentralized dapp in the Polkadot ecosystem on [Github](https://github.com/topics/polkadot). You can read more about their work and KodaDot's contributions to the Polkadot ecosystem on the [Polkadot Wiki](https://wiki.polkadot.network/docs/learn-nft-projects#kodadot).
+
+### Team Code Repos
+
+- [https://github.com/kodadot/stick](https://github.com/kodadot/stick)
+- [https://github.com/kodadot/nft-gallery](https://github.com/kodadot/nft-gallery)
+
+#### Team GitHub accounts 🧑‍💻
+
+- [Viktor Valaštín](https://github.com/vikiival)
+- [Matej Nemček](https://github.com/yangwao)
+
+### Team LinkedIn Profiles 🧑‍🎓
+
+- [Viktor Valastin](https://linkedin.com/in/vikival/)
+- [Matej Nemcek](https://linkedin.com/in/mnemcek/)
+
+
+## Development Status :open_book:
+
+- [github.com/kodadot/stick/milestone/3](https://github.com/kodadot/stick/milestone/3) - Milestone 2 track
+- [github.com/kodadot/stick](https://github.com/kodadot/stick) This is the repo for AssetHub Indexer.
+- [github.com/kodadot/uniquery](https://github.com/kodadot/uniquery) This is the repo for SDK.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Estimated duration:** 3 months ⌛️
+- **FTE:** 2 FTE
+- **Costs:** 30,000 USD 💰
+
+## Milestone 2 - AssetHub Indexer Swaps, Offers and Tokens Implementation
```

Comment:
Thanks for the application, @vikiival. You have shown that there is an active target audience for the indexer and that you have thought the work through. Just one request, please rename it to Milestone 1, as we rely on the numbering for tracking project progress. :)

```suggestion
## Milestone 1 - AssetHub Indexer Swaps, Offers and Tokens Implementation
```